api/target.go

```diff
@@ -0,0 +1,306 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/gin-gonic/gin"
+	"github.com/konveyor/tackle2-hub/model"
+	"gorm.io/gorm/clause"
+	"net/http"
+)
+
+//
+// Routes
+const (
+	TargetsRoot = "/targets"
+	TargetRoot  = TargetsRoot + "/:" + ID
+)
+
+//
+// TargetHandler handles Target resource routes.
+type TargetHandler struct {
+	BaseHandler
+}
+
+func (h TargetHandler) AddRoutes(e *gin.Engine) {
+	routeGroup := e.Group("/")
+	routeGroup.Use(Required("targets"), Transaction)
```

Comment:
This is probably practical but expected the Transaction only on:
Create
Update
Delete

Reply:
I think we've been using it like this everywhere, and relying on the method check in the Transaction handler unless I'm mistaken.

Reply:
You're correct.  I forgot.